tests/test_blueprint.py

```diff
@@ -570,6 +570,23 @@ def test_2():
         )
         assert spec["paths"]["/test/test_2/"]["get"]["tags"] == ["test", ]
 
+    def test_blueprint_route_hidden_from_api_spec(self, app):
+        """Check passing hidden_from_api_spec to route"""
+        blp = Blueprint('test', __name__, url_prefix='/test')
+
+        @blp.route('/test_1/', hidden_from_api_spec=True)
+        def test_1():
+            pass
+
+        @blp.route('/test_2/')
+        def test_2():
+            pass
+
+        api = Api(app)
+        api.register_blueprint(blp)
+        spec = api.spec.to_dict()
+        assert (len(spec["paths"]) == 1)
```

Comment:
I'd test this instead of the length.

```py
        assert "/test/test_1/" not in spec["paths"]
        assert "/test/test_2/" in spec["paths"]
```

Reply:
Ohh alrighty, learned something new. I leave the requirements file for what it is.

I applied the changes for the unittest locally, would you like me to commit it all in one commit or would you like me to make 2 commits. One for docstring and one for unittest?

Reply:
Ultimately, you should learn the magic of `rebase -i`. It comes in very handy to rewrite history.

For now, the easiest path for you if you're not comfortable with squasing is to amend your last commit then push force.

```
git add .
git commit --amend
git push -f
```

Reply:
Yes, you are right. I might actually just make a repository to play around with this. --amend was also new for me.

Thank you for your input, much appriciated!

I've committed the changes : )